assets/src/edit-story/components/panels/test/actions.js

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
The Actions panel was really a temporary panel until removing an element via keyboard was implemented (which should be done now). Both the action panel and the tests for it should be obsolete.

(Can be removed elsewhere out of this PR though)